CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- added in colour variables for discovery and live badges and banners
+- updated link colour variables to match gov.uk [#795](https://github.com/hmrc/assets-frontend/pull/795)
```

Comment:
@rpowis seeing as we are trying to promote consistency we should probably align to the GOV.UK content style for bullets, which is to use lowercase https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps